README-group.md

```diff
@@ -130,6 +130,23 @@ And ensure the presence of the groups with this example playbook:
       groups: "{{ groups }}"
 ```
 
+Example playbook to rename a group:
+
+```yaml
+---
+- name: Playbook to handle users
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  # Rename group appops to webops
+  - ipagroup:
```

Comment:
Using named tasks in example playbooks could enhance clarity.

Reply:
Good catch!

Reply:
Fixed.